_data/sites.json

```
@@ -4506,6 +4542,16 @@
         ]
     },
 
+    {
+            "name": "Copyleaks",
+            "url": "https://id.copyleaks.com/security",
+            "difficulty": "medium",
+            "notes": "Account > Security > Delete Account (On the bottom of the page) for all data: https://id.copyleaks.com/security/delete-internal-data.",
```

```suggestion
            "notes": "Account ⮕ Security ⮕ Delete Account (On the bottom of the page). If you wish to delete all data, use [this page](https://id.copyleaks.com/security/delete-internal-data).",
```

We use `⮕` as arrows across the whole of JSON and tend to wrap links into the markdown hyperlink syntax so that it looks neater on the web page.